tests/globalkilltest/global_kill_test.go

```diff
@@ -487,11 +577,16 @@ func (s *TestGlobalKillSuite) TestLostConnection(c *C) {
 	c.Assert(err, NotNil)
 	c.Assert(err.Error(), Equals, "driver: bad connection")
 
-	// start PD proxy to restore connection.
-	log.Infof("restart pdProxy")
-	pdProxy1, err := s.startPDProxy()
+	err = s.tikvProc.Process.Kill()
+	c.Assert(err, IsNil)
+	err = s.tikvProc.Wait()
+	if err != nil {
+		c.Assert(err.Error(), Equals, "exec: Wait was already called")
+	}
+	time.Sleep(1 * time.Second)
```

Comment:
Why we need sleep 1 second..?

Reply:
I added it to stable the test, but it seems redundant, and I'll remove it.